src/corpus/corpus.cpp

```diff
@@ -3,9 +3,9 @@
  * @author Sean Massung
  */
 
+#include "meta/corpus/corpus.h"
 #include "cpptoml.h"
 #include "meta/corpus/all.h"
-#include "meta/corpus/corpus.h"
```

Comment:
It puts the file's own header first. Believe the order is

System
Self
Others

All respectively alphabetized.